docs/Guides/Recommendations.md

```diff
@@ -298,3 +299,28 @@ readinessProbe:
     timeoutSeconds: 3
     successThreshold: 1
     failureThreshold: 5
+```
+
+## Capacity planning for production
```

Comment:
This needs to pass a message of "do your own measurements".

Reply:
@mcollina Adjusted. Do you think that the emphasis is correct now?